src/Model/LinkBlock.php

```diff
@@ -63,7 +63,7 @@ class LinkBlock extends \ObjectModel
         'primary' => 'id_link_block',
         'multilang' => true,
         'fields' => array(
-            'name' => array('type' => self::TYPE_STRING, 'lang' => true, 'required' => true, 'size' => 128),
+            'name' => array('type' => self::TYPE_STRING, 'lang' => true, 'required' => true, 'size' => 40),
```

Comment:
Should it be in an upgrade script?
What about values already higher than 40?

Reply:
From what I see this value has no impact on module anyway, and DB has 40 chars limit.

Reply:
I double checked it, it should be fine :D People who tried to add something > 40 chars had this fatal error from issue